src/common/hooks/useConfig.hook.ts

```diff
@@ -26,13 +21,13 @@ type GetProfiles = {
 
 export const useConfig = () => {
   const {
+    schemaCreatorService: baseSchemaCreatorService,
```

Comment:
Why do we need to invoke these with an alias only to reassign to a variable with the name that's identical to the initial one?